web/src/lib/components/pages/record.tsx

```diff
@@ -187,10 +187,12 @@ export default class RecordPage extends Component<RecordProps, RecordState> {
 
     // Don't start a new recording when full.
     } else if (!this.isFull()) {
-      this.audio.init().then(() => {
-        this.startRecording();
-      });
-    }
+      	this.audio.init().then(() => {
+        	this.startRecording();
+      	}).catch((err) =>{
+      		window.location.href = "/";
```

Comment:
This will cause the page to reload, which is not what we want (that's why we filed #233 in the first place). Best to just do nothing when the users cancels.

Also please follow code conventions. Looks like you are using tabs, whereas we have chosen spaces.